packages/zwave-js/src/lib/driver/ZWaveOptions.ts

```diff
@@ -53,12 +53,19 @@ export interface ZWaveOptions {
 	 */
 	logConfig?: LogConfig;
 
-	/**
-	 * @internal
-	 * Set this to true to skip the controller interview. Useful for testing purposes
-	 */
-	skipInterview?: boolean;
+	interview: {
+		/**
+		 * @internal
+		 * Set this to true to skip the controller interview. Useful for testing purposes
+		 */
+		skipInterview?: boolean;
 
+		/**
+		 * Whether all user code should be queried during the interview of the UserCode CC.
+		 * Note that enabling this can cause a lot of traffic during the interview.
+		 */
+		queryAllUserCodes?: boolean;
+	};
```

Comment:
this needs to be added to `8.0.0` migration notes as a breaking change. Not sure if there's a process for this?

Reply:
Last time I added a migration guide in the docs and extended it in each PR.

Reply:
If that's ok with you, I will add a `migrating-to-v8.md` file now that follows roughly the structure of prior file. I will then add this change in zwave options to it.

Reply:
e.g. https://github.com/zwave-js/node-zwave-js/pull/2932/commits/fd4ad42b7a5fcb6ea4446920650c7668b6b6a92c